news/2018-02-05-project-loved-february-week-1.md

```diff
@@ -0,0 +1,183 @@
+
+Their style cannot be understated, as they take all the most unique parts of a song and end up well-representing them in a map, one way or another. Figue Folle is no different, and we enjoy it for those quirks and qualities as well.
+
+[![](/wiki/shared/news/2018-02-05-project-loved-february-week-1/osu!taiko/sayonara-planet-wars.jpg)](https://osu.ppy.sh/community/forums/topics/699495)
+
+Ah, a funny little quirk about this map. The description for this actually says "This WILL be my first ranked mapset, or else I am not BrambleClaw!". Well, [BrambleClaw](https://osu.ppy.sh/u/3487731) actually changed his name to Bramble, and this will possibly be his first Loved mapset. 
+
+Kinda funny how life works isn't it? Anyway, a map well known to players from TWC 2016, this is a very fun set with a little for all types of players, with an outstanding top diff.
+
+### <a name="osumania" id="osumania"></a>osu!mania
+
+[![](/wiki/shared/news/2018-02-05-project-loved-february-week-1/osu!mania/the-mastermind.jpg)](https://osu.ppy.sh/community/forums/topics/699501)
+
+The Mastermind is not just a fancy title; the entire chart is a battle against what you see and what you're actually supposed to do. 
+
+This chart contains blisteringly fast but very discreet SVs in some sections, and extremely clinical slowdowns in certain others. This creates an interesting metagame element; do you want to reduce your speedmod to deal with the super fast SVs and in return deal with even slower slowdowns, or do you want to increase your speedmod to make the slowdowns a little easier at the expense of increasing the probability of messing up in the super fast SV sections? 
+
+This additional level of metagame, unique chart structure, and a chart that reflects the callousness and unforgiving nature of the track are reasons why I believe that many players will receive it very well (and some of them do already) and why this chart should be in the Loved section.
+
+[![](/wiki/shared/news/2018-02-05-project-loved-february-week-1/osu!mania/souzou-forest.jpg)](https://osu.ppy.sh/community/forums/topics/699502)
+
+Souzou Forest is one of [17VA](https://osu.ppy.sh/u/1895984)'s most popular charts, it is a fun LN chart which is decently fast but also fairly simplistic stair based patterning, which makes it both easy to hit and hard to get good accuracy on. 
+
+It has been featured in the MWC 7K 2016 Quarterfinals and has retained its popularity to this day.
+
+[![](/wiki/shared/news/2018-02-05-project-loved-february-week-1/osu!mania/the-scales-of-struggle.jpg)](https://osu.ppy.sh/community/forums/topics/699503)
+
+As the song title suggests, The Scales of Struggle scales the amount of struggle with each step up the difficulty ladder. Ranging from easy peasy to outright warfare, each difficulty tests the waters of speedy technicality for the whole playerbase.
+
+A great collaboration between veteran charter Fullerene- and veteran speedster Staiain, which we're sure you'll enjoy.
```

Comment:
Fullerene- (https://osu.ppy.sh/u/2531335) and Staiain (https://osu.ppy.sh/u/86188) profiles should be added